blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/serializers/NumericSerializer.java

```diff
@@ -71,6 +66,7 @@
         static final byte B_TIMER = (byte)'T';
         static final byte B_SET = (byte)'S';
         static final byte B_GAUGE = (byte)'G';
+        static final byte B_ENUM = (byte) 'E';
```

Comment:
NIT:  should remove the white space between the cast and the letter like the other constants above.